docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.7'
+services:
+  replication-solr:
+    image: cnxta/replication-solr:0.3.0-SNAPSHOT
+    ports:
+     - "8983:8983"
+    volumes:
+      - replication-data:/var/solr
+    networks:
+      - replication
+    entrypoint:
+      - bash
+      - "-c"
+      - "precreate-core replication_item; precreate-core replication_site; precreate-core replication_status; precreate-core replication_config; exec solr -f"
+  ion-replication:
+    image: cnxta/replication:0.3.0-SNAPSHOT
+    networks:
+      - replication
+    entrypoint:
+      - bash
+      - "-c"
+      - "java \
+        -Dssl.system.properties=/opt/replication/ssl.properties \
+        -Djavax.net.ssl.keyStore=/opt/replication/serverKeystore.jks
+        -Djavax.net.ssl.trustStore=/opt/replication/serverTruststore.jks
+        -jar /opt/replication/app.jar --spring.config.location=file:/opt/replication/"
+    secrets:
+      - source: replication-keystore
+        target: /opt/replication/serverKeystore.jks
+      - source: replication-truststore
+        target: /opt/replication/serverTruststore.jks
+      - source: replication-ssl
+        target: /opt/replication/ssl.properties
+    configs:
+      - source: replication-spring-config
+        target: /opt/replication/application.properties
+configs:
+  replication-spring-config:
+    external: true
+secrets:
+  replication-keystore:
+    external: true
+  replication-truststore:
+    external: true
+  replication-ssl:
+    external: true
+volumes:
+  replication-data:
+    external: true
```

Comment:
If we make remove this line and the one below it there will be no need to manually create the volume using the `docker volume create` command

Reply:
removed